src/components/RichEditor/index.jsx

```diff
@@ -103,8 +128,9 @@ class RichEditor extends React.Component {
               content_style: contentStyle,
               default_link_target: '_blank',
             }}
-            onChange={this.updateCharCount}
-            onKeyUp={this.updateCharCount}
+            onChange={this.handleOnChange}
+            onEditorChange={this.handleEditorChange}
```

Comment:
[curious] Why has this been added here?

Reply:
It is already explained in PR description. To summarize, onEditorChange is tinymce's own event for detecting content changes and is recommended to be used, especially when making the editor a controlled component.